databricks/koalas/series.py

```diff
@@ -958,13 +958,11 @@ def rename(self, index: Union[str, Tuple[str, ...]] = None, **kwargs):
     def index(self):
         """The index (axis labels) Column of the Series.
 
-        Currently not supported when the DataFrame has no index.
-
         See Also
         --------
         Index
         """
-        return self.to_frame().index
```

Comment:
Can you just change this to `self._kdf.index`?

Reply:
@HyukjinKwon ah, yes i can. thanks! :)